client/signup/config/flows-pure.js

```diff
@@ -422,6 +422,13 @@ export function generateFlows( {
 			providesDependenciesInQuery: [ 'siteSlug' ],
 			pageTitle: translate( 'Setup your site' ),
 		},
+		{
+			name: 'built-by-wpcom',
```

Comment:
Would it be possible to use a flow name that's closer to the product name? The product name is Do It For Me, so may I suggest do-it-for-me?

Built By WordPress.com Premium is the Premium product

Reply:
Done :)